pkg/kubelet/dockertools/manager.go

```diff
@@ -511,6 +511,9 @@ func (dm *DockerManager) runContainer(pod *api.Pod, container *api.Container, op
 	if len(opts.DNSSearch) > 0 {
 		hc.DNSSearch = opts.DNSSearch
 	}
+	if len(opts.CgroupParent) > 0 {
```

Comment:
Yep. I tested with Docker 1.4.1. Containers just launch under the default `/docker` cgroup.